### DISCUSSION TITLE
Restore additional log bases for graphing

### DISCUSSION THREAD

**Discussion by virusdave** [grafana/grafana#65580] [Dashboards, Panels & Visualizations]
The [ancient FR](https://github.com/grafana/grafana/issues/452) for logarithmic Y-axis was closed with (what appeared to be) support for [additional log bases](https://github.com/grafana/grafana/commit/36a948965bd1051ea885434dcdf2e0f357d9a3ca) beyond just 2 and 10.

Somehow, this functionality appears to have been lost.  Log base 2 is fine, or rather, _would be_ fine if Y-axis gridlines were intelligently chosen to be readable, by dropping gridline labels that overlap each other.  Alternatively, allow user specification of the log base, then continue placing one gridline per order-of-base.

Example to illustrate the problem:
<img width="790" alt="image" src="https://user-images.githubusercontent.com/6148271/228697876-614ebd9d-082a-46f9-bdf8-e541b1387184.png">

The left Y-axis is useless as-is, since the graphed series cross roughly 25 powers of two (32b/s min vs ~1Gbps max).
Intelligent selection of Y-axis gridline labels might have chosen between 4 and 8 labels (both [32b/s, 1Kib/s, 32Kib/s, 1Mib/s, 32Mib/s, 1Gib/s] and [1b/s, 1Kib/s, 1Mib/s, 1Gib/s] would have probably been reasonable choices).  Had I been able to choose, I would have selected a "base-32 logarithmic" graph for this purpose, although honestly just being intelligent in labelling and sticking with base-2 would probably be better.

**Comment by leeoniya** [comment]
i converted this to an issue. let's track it in https://github.com/grafana/grafana/issues/65581
